packages/gap-web-ui/src/components/notification-banner/ImportantBanner.tsx

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 
 /**
  * Generic component to be used when adding a GDS important notification banner.
- * Currently we provide bannerContent prop for single line notifications,
- * if this component needs to use more than a single line,
- *  it is recommended to restructure it to use a h3 and a p tag
  */
-const ImportantBanner = ({ bannerContent }: ImportantBannerProps) => {
+const ImportantBanner = ({
+  bannerHeading,
+  bannerContent,
+  successBanner = false,
```

Comment:
would suggest renaming this for clarity: 

```suggestion
  isSuccess = false,
```